libs/community/tests/integration_tests/document_loaders/parsers/test_images.py

```diff
@@ -18,12 +20,13 @@
 text_image = Blob.from_path(path_base / "examples/text.png")
 page_image = Blob.from_path(path_base / "examples/page.png")
 
+_re_in_image = r"(?ms).*MAKE.*TEXT.*STAND.*OUT.*FROM.*"
```

Comment:
Thanks for the fix!

The testing code is fairly difficult to understand. It would be nice to simplify the test logic to make it easier to maintain

It uses two layers of parameterization and also a regexp that does not seem necessary.